v1/evidence/artifact-evidence-exists.yaml

```diff
@@ -0,0 +1,6 @@
+name: artifact-evidence-exists
+description: "A rule to verify that the artifact produced any evidence"
+labels:
+  - sample-policy-bundle
+initiatives:
+  - client-policy
```

Comment:
It seems we cannot have an empty `format-type`, so we need to set it by default (I set it to `cyclonedx-json` in `v1/sboms/evidence-exists.yaml`).